--- data/egg/f3_baseline/SCHEDULE_NEW.INC	2019-11-05 10:29:38
+++ data/egg/f3_perturbed/SCHEDULE_NEW.INC	2026-09-22 16:58:00
@@ -1,256 +1,256 @@
-WCONINJE
-'INJECT1'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT2'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT3'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT4'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT5'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT6'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT7'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-'INJECT8'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
-/
-
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-TSTEP
-   30 /
-
-END
-
+WCONINJE
+'INJECT1'	'WATER'	'OPEN'	'RATE'	60.0 1* 420/
+'INJECT2'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT3'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT4'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT5'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT6'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT7'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+'INJECT8'	'WATER'	'OPEN'	'RATE'	79.5 1* 420/
+/
+
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+TSTEP
+   30 /
+
+END
+
